cabal-install/tests/UnitTests/Distribution/Solver/Modular/Solver.hs

```diff
@@ -961,7 +961,7 @@ tests =
                     , Right $ exAv "B" 1 [ExFix "A" 4]
                     ]
                   rejecting = "rejecting: A-3.0.0/installed-3.0.0"
-                  skipping = "skipping: A; 2.0.0/installed-2.0.0, 1.0.0/installed-1.0.0"
+                  skipping = "skipping: A; 1.0.0/installed-1.0.0, 2.0.0/installed-2.0.0 "
```

Comment:
What's the source of the trailing space?

Reply:
Its un-needed. Removed it